### DISCUSSION TITLE
真鍮のランタンをゲーム開始直後に確実に入手できるようにしてはどうか

### DISCUSSION THREAD

**Discussion by chikatei** [hengband/hengband#400] [Closed] [answered]
ゲーム開始直後、真鍮のランタンが雑貨屋に置いていない場合にQy@したり時間を経過させて店を覗き直すのは単に作業になってしまっている気がします。序盤の進行をスムーズにする更新の一環として、ランタンが確実に入手できるようにしてみるのはどうでしょうか。
確実にランタンを入手させる方法としては、たとえば
・雑貨屋で必ず売られるようにする
・@の初期所持品に加える
・短愚蛮怒のように証人クエ内に配置する
などが考えられると思います。

もっとも、「ランタンが売ってないからQy@」というのは伝統や様式美の域に達した定番ネタなので、聖域として残しておくのも十分ありだと思いますが、実際に面倒くさいと思ってしまうときもあるので、ランタンの確定入手という仕様を提案してみます。

自分では明確には思い浮かばなかったのですが、逆にランタンが確定入手となることによってゲームとしての楽しみが損なわれる、といった意見などがありましたら現状のままにすべきかもしれませんし、この辺の是非については他の方にも聞いてみたいです。

**Comment by ghost** [comment]
プレーして日が浅いので歴史的経緯とか知らないのですが、初心者ガイドに下記の記述があります。
ランタン初期装備にするならこの辺もメンテしてあげてくださいとだけ。
>1番と2番の店、雑貨店と防具屋に立ち寄るのを忘れないで下さい。雑貨屋で  は、
>クロークと真鍮のランタン、そして油つぼを1～2本買います。
>もともと持っていた松明を売ってしまいましょう。
>中略
>もちろん、これらのアイテムが売られていないこともあります。
>それぞれの店は時々店のアイテムを新しく入れ替えます。
>もし、あなたが必要としているものが現状で売られていなければ、店の品物が入れ替わるのを待つこともできます。

なおランタンがないと証人クエで事故りやすいので初心者救済したいという観点であれば、松明を初期所持品としたうえで
　・クエストマップを最初からライトエリア済にする
といった対応で救済しつつ様式美を守ることも可能ではと思いました。

**Comment by deskull-m** [comment]
伝統といってもなんですし、ランタン含めてまともな必需品常備の方向性でいきたいですねえ。もうちょっと意見見てそうIssue化します。

**Comment by iks3** [comment, marked as answer]
α10で店売りの調整が終わり、ランタンは常時販売になりました。

**Reply by deskull-m** [reply]
調整ありがとうございました。
